src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,49 +1,108 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private final int startLength = 10;
```

Comment:
```suggestion
    private static final int DEFAULT_CAPACITY = 10;
```
краще такі змінні робити константами ( static final)

Reply:
+